app/components/back_link_component.rb

```diff
@@ -0,0 +1,6 @@
+class BackLinkComponent < ApplicationComponent
+  def initialize(title:, link:)
```

Comment:
By having this, you are allowing nil to be a valid value. We want it to not work unless there's values passed. Are there reasons to keep it nil?

Reply:
If you default to nil, and use the `render?` method, you let the component be in charge of choosing to render or not. This way if someone screws up and it slips to production, it will simply just not render rather than causing an error that prevents the page from loading. Something I’ve also been considering is raising in render only in development if they aren’t passed in. 

```ruby
def render?
  raise ArgumentError.new("Title and link cannot be nil") if Rails.env.development && (title.nil? || link.nil?)
  title && link
end 
```